src/routes/events/$eventId.tsx

```
@@ -38,22 +42,92 @@ const styles = {
+      try {
+        const currentUser = await userController.getCurrentUser();
+        if (!currentUser || !currentUser.id) {
+          setIsFavorite(false);
+          return;
+        }
+        
+        const userFavoritesList = await favoriteController.getFavorites("uuid, event_id");
+        if (userFavoritesList && userFavoritesList.length > 0) {
+          const userFavoriteEntry = userFavoritesList.find(fav => fav.id === currentUser.id);
+
+          if (userFavoriteEntry && userFavoriteEntry.event_id && userFavoriteEntry.event_id.includes(event.id)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)